### PR TITLE
DOC: Describe speclite filters in documentation

### PR DESCRIPTION
**Pull request by philipp128**
## Description
Addresses Issue #444. This PR adds a small description where to find the spectroscopic filter bands available as an input for the functions of the [Photometry module](https://github.com/skypyproject/skypy/blob/main/skypy/utils/photometry.py).

I added a link to the corresponding speclite documentation in the docs of the `Photometry` module.

## Checklist
- [ ] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/main/CONTRIBUTING.rst)
- [ ] Assign someone from the infrastructure team to review this pull request

### CODE REVIEW

File: docs/utils/index.rst [hunk 1]
```diff
@@ -126,6 +126,15 @@ Photometry (`skypy.utils.photometry`)
...
 
+Some of the following functions require a photometric filter as an input
+parameter. In SkyPy, we included the
+`speclite <https://speclite.readthedocs.io/>`_ package to enable different
+spectroscopic tasks
+(see `Dependencies <https://skypy.readthedocs.io/en/latest/install.html#dependencies>`_).
+One can choose between the available filters described
+`here <https://speclite.readthedocs.io/en/latest/filters.html>`_ by using the
+same syntax (see `speclite.filters.load_filters`).
+
```
Comment (rrjbca) [inline docs/utils/index.rst:137]:
```suggestion
SkyPy uses the `speclite <https://speclite.readthedocs.io/>`_ package for
photometric calculations. Some of the following functions take the names of
photometric filters as an input parameter. Users can choose from the available
`Speclite Filters <https://speclite.readthedocs.io/en/latest/filters.html>`_
following the naming syntax described in `speclite.filters.load_filters`, or
create their own named `speclite.filters.FilterResponse`.

```

Comment (rrjbca) [reply docs/utils/index.rst:137]: Suggestion for clarity and also to highlight that users can create their own `FilterResponse`

**Comment by rrjbca** [comment]
Is this the best/only place to provide this information? It might be sensible to duplicate everything here: https://skypy.readthedocs.io/en/latest/galaxies.html#spectrum as this is where the user-facing functions (e.g. kcorrect) are documented.

**Comment by philipp128** [comment]
> Is this the best/only place to provide this information? It might be sensible to duplicate everything here: https://skypy.readthedocs.io/en/latest/galaxies.html#spectrum as this is where the user-facing functions (e.g. kcorrect) are documented.

I was thinking about the same. I was not quite sure where the best place for this is. I think we definitely should keep it where it is now as this refers to the functions which have the filters as an input. 

Let me also add it to the suggested location.

**Review by rrjbca — APPROVED** [review]

**Pull request merged**